content/Day-8/_index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: 8. Technical Analysis Templates
+weight: 108
+---
+
+## Knowing What To Include
+***
+When we create our technical documentation we need to know which parts are the sum parts that make up the whole document. The first step is to determine what we are writing about. Then we need to determine who is the target audience. Once we have these two ideas defined, we can move towards creating an outline and gathering requirements.   
+
+## Gather Requirements 
+
+Requirements gathering is the process of collecting information about what needs to be done. We gather information from stakeholders. A stakeholder can be anyone who has interest in the project we are working on. They can be members on the leadership team or colleagues that work alongside us. More often than not, they will be project managers or technical team leaders. 
+
+## Visual Aids
+
+For complex designs and concepts ideas we should accompany the design with illustrations (drawings) to help the readers understand the complexity within the system.  This is common in web application architecture. For our role as technical analyst, this includes screenshots, conversation summary, steps taken by us and any referenced documentation that was used to help troubleshoot the reported problem. These are effectively your notes. Our notes should stand on their own without support. We do not know who will review our notes or what part of the world they may be in when they find our notes. We should include all the steps required to carry the problem forward with repeating unnecessary steps. 
```

Comment:
"concepts ideas" should maybe be just concepts

Reply:
This content has been modified/updated